public/js/Histogram.js

```diff
@@ -12,21 +16,28 @@ _.mixin(require('lodash-deep'));
  * @param {array} featureNames - An array of the features used in this screen.
  */
 function Histogram(sampleData, element, featureNames) {
+    var self = this;
+
     this.sampleData = sampleData;
     this.featureNames = featureNames;
     this.feature = 0;
-
     this.element = element;
-    this.fullWidth = 500;
-    this.fullHeight = 400;
+
+    this.fullWidth = $(this.element).width();
```

Comment:
Good God! I finally get it! This is even more egregious than lodash! LOL

What about `$foo`? Does that still "call" jQuery? (ie parens are optional?)